app/controllers/single_page_subscriptions_controller.rb

```diff
@@ -0,0 +1,58 @@
+class SinglePageSubscriptionsController < ApplicationController
+  include AccountHelper
+  include GovukPersonalisation::ControllerConcern
+
+  before_action do
+    head :not_found unless govuk_account_auth_enabled?
+  end
+
+  def show
+    content_item = GdsApi.content_store.content_item(topic).to_h
+    if logged_in?
+      subscriber = GdsApi.email_alert_api.link_subscriber_to_govuk_account(
+        govuk_account_session: @account_session_header,
+      )
+
+      subscriber_list = GdsApi.email_alert_api.find_or_create_subscriber_list(
+        {
+          url: topic,
+          name: content_item["title"],
+          content_id: content_item["content_id"],
+        },
+      )
```

Comment:
Does this need the parameters to be passed in a hash?  That's not the convention other things use.

Reply:
I think this is correct, unless I'm reading the [definition of `find_or_create_subscriber_list`](https://github.com/alphagov/gds-api-adapters/blob/91639289645173f92f4367f7d53f00e51957bc30/lib/gds_api/email_alert_api.rb#L9-L20) very wrong. 

It takes a single parameter `attributes` which is a hash that includes one of `content-id`, `links` or `tags`, but not more than one.